### PR TITLE
Add task to migrate from s3 to local storage

### PR DESCRIPTION
**Pull request by tramuntanal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [0.1.1] - 2023-04-27 (Una mascota trapellota)
```
Comment (laurajaime) [inline CHANGELOG.md:3]: 😂

File: README.md [hunk 2]
```diff
@@ -48,6 +48,23 @@ To execute the task run:
 bin/rake cdtb:fix_nicknames
 ```
 
+#### Migrate ActiveStorage service from S3 to local
+
+To migrate from S3 to local storage, the identified steps will be:
+
+1. Download the assets to a temporary directory:
+    `aws s3 sync s3://bucket-name tmp/storage/`
+2. Move the downloaded assets into the local storage directory doing the sarding:
```
Comment (laurajaime) [inline README.md:57]:
```suggestion
2. Move the downloaded assets into the local storage directory doing the sharding:
```

File: README.md [hunk 3]
```diff
@@ -48,6 +48,23 @@ To execute the task run:
...
+    `bin/rake cdtb:s3_to_local:do_sharding`
+3. Update all blobs to use the local service
+    `bin/rake cdtb:s3_to_local:set_local_service_on_blobs`
+4. Clean the cache:
+    `bin/rake cache:clear`
+5. Restart the Rails server
+
+
+
+
```
Comment (laurajaime) [inline README.md:67]: Can you remove the extra spaces, please? 😅

File: lib/decidim/cdtb/task.rb [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        log_task_end
+      end
+
+      #################################
+
+      protected
+
+      #################################
+
+      # May be used by sublasses for preparing before executing the task
```
Comment (laurajaime) [inline lib/decidim/cdtb/task.rb:49]:
```suggestion
      # May be used by subclasses for preparing before executing the task
```

File: lib/decidim/cdtb/task.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+      def prepare_execution(context); end
+
+      # Sublasses must implement the steps of the task overriding this method.
+      def do_execution(context); end
+
+      # May be used by sublasses for doing whatever after executing the task
```
Comment (laurajaime) [inline lib/decidim/cdtb/task.rb:55]:
```suggestion
      # May be used by subclasses for doing whatever after executing the task
```

**Review by laurajaime — CHANGES_REQUESTED** [review]
Good job!! 

Only some details.

**Comment by tramuntanal** [comment]
Thanks for the review, corrections applied! :eyes:

**Pull request merged**